tools/topology/topology2/cavs/platform/intel/hdmi-4th.conf

```diff
@@ -30,6 +31,7 @@ Object.Pipeline {
 			dai_type "HDA"
 			copier_type "HDA"
 		}
+		use_chain_dma	$USE_CHAIN_DMA
```

Comment:
@ranj063  if chain dma is enabled, do we need to disable S24_LE in pcm_caps ?

Reply:
@RanderWang AFAIK there's no restriction for not having S24_LE with chained DMA. I know it doesnt work well today but we need to fix it

Reply:
ok looks like we do have the restriction. I will remove S24_Le with chained DMA